src/usdb_syncer/notes_parser.py

```diff
@@ -40,17 +40,24 @@ class Note:
 
     @classmethod
     def parse(cls, value: str) -> Note:
-        regex = re.compile(r"(:|\*|F|R|G):? +(-?\d+) +(\d+) +(-?\d+) (.+)")
+        regex = re.compile(r"(:|\*|F|R|G):? +(-?\d+) +(\d+) +(-?\d+)(?: (.*))?")
         if not (match := regex.fullmatch(value)):
             raise NotesParseError(f"invalid note: '{value}'")
         text = match.group(5)
+        if not text:
+            text = ""
```

Comment:
You can also write `text = match.group(5) or ""`. (I hope pylint understands that as well.)